hash/jaguar.xml

```diff
@@ -180,11 +195,14 @@ Unreleased (possibly no prototypes exist):
 		</part>
 	</software>
 
-	<!-- boot OK, title screen foreground layer not drawn properly, no sound -->
 	<software name="mutntpng" supported="partial">
 		<description>Attack of the Mutant Penguins</description>
 		<year>1996</year>
 		<publisher>Atari</publisher>
+		<notes><![CDATA[
+Boot OK, title screen foreground layer not drawn properly, no sound
```

Comment:
As already stated in a previous PR we decided to ditch "boot OK" from notes.

Reply:
Removed "boot OK" from `notes`